includes/admin/upgrades/v3/class-data-migrator.php

```diff
@@ -770,8 +770,16 @@ public static function orders( $data = null ) {
 			) );
 		}
 
-		// By default, this is what is stored in payment meta.
-		$core_meta_keys = array(
+		/**
+		 * By default, this is what is stored in payment meta. These array keys are part of the core payment meta in 2.x
+		 * but are not needed as part of the order meta.
+		 * Extensions can add their keys to this filter if they use the payment meta array to store data and have
+		 * established a migration process to keep the data intact with the new order tables.
+		 *
+		 * @since 3.0
+		 * @param array The array of payment meta keys.
+		 */
+		$core_meta_keys = apply_filters( 'edd_30_core_payment_meta', array(
```

Comment:
Can we rename the filter? Although it makes sense with how _we're_ using it, the point of the filter is to support third parties, in which case it's no long really "core" payment meta. Maybe something like:

`edd_30_payment_meta_to_exclude` or `edd_30_payment_meta_keys_not_migrated`

:shrug: Something like that. Just to better convey what actually happens to those keys (or _doesn't_ happen).